linode_api4/objects/linode.py

```diff
@@ -764,20 +784,48 @@ def kvmify(self):
 
         return True
 
-    def mutate(self):
+    def mutate(self, allow_auto_disk_resize=True):
         """
         Upgrades this Instance to the latest generation type
         """
-        self._client.post('{}/mutate'.format(Instance.api_endpoint), model=self)
+
+        params = {
+            "allow_auto_disk_resize": allow_auto_disk_resize
+        }
+
+        self._client.post('{}/mutate'.format(Instance.api_endpoint), model=self, data=params)
 
         return True
 
-    def initiate_migration(self):
+    def initiate_migration(self, region=None, upgrade=None):
         """
         Initiates a pending migration that is already scheduled for this Linode
         Instance
         """
-        self._client.post('{}/migrate'.format(Instance.api_endpoint), model=self)
+        params = {
+            "region": region.id if issubclass(type(region), Base) else region,
+            "upgrade": upgrade
+        }
+
+        self._client.post('{}/migrate'.format(Instance.api_endpoint), model=self, data=params)
```

Comment:
Could we use the `drop_null_keys` helper to remove any unspecified arguments from the API request? https://github.com/linode/linode_api4-python/blob/dev/linode_api4/util.py#L7